screengrab/README.md

```diff
@@ -35,6 +35,8 @@ screengrab
 
 ###### Automated localized screenshots of your Android app on every device
 
+Generate localised Android screenshots for different device types and languages for Google Play that are ready to be uploaded using ([`supply`](https://github.com/fastlane/fastlane/tree/master/supply).
```

Comment:
`screengrab` generates localized screenshots of your Android app for different device types and languages for Google Play and can be uploaded using [`supply`](https://github.com/fastlane/fastlane/tree/master/supply).